### PR TITLE
Fix formatting via PHP CS Fixer

### PR DESCRIPTION
**Pull request by junichi11**
- Run GitHub Actions on push and pull_request

### CODE REVIEW

File: src/Config.php [hunk 1]
```diff
@@ -3,109 +3,128 @@
 use utils\Log;
 use utils\Strings;
 
-final class Config {
-
+final class Config
+{
```
Comment (tmysik) [inline src/Config.php]: Uff, do we really want to:
- use this formatting?
- loose Git history?

**Review by tmysik — COMMENTED** [review]
@junichi11 Please, see my question. Cannot we configure PHP CS Fixer to use _Java coding style_?

**Comment by junichi11** [comment]
> Please, see my question. Cannot we configure PHP CS Fixer to use _Java coding style_?

@tmysik I'll use it if you prefer it, of course :) I'll fix the configuration file. Thanks!

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        'return_type_declaration' => true,
+        'single_blank_line_at_eof' => true,
+        'single_blank_line_before_namespace' => true,
+        'single_class_element_per_statement' => true,
+        'single_import_per_statement' => true,
+        'single_line_after_imports' => true,
+        'switch_case_semicolon_to_colon' => true,
+        'switch_case_space' => true,
+        'ternary_operator_spaces' => true,
+    ])
```
Comment (junichi11) [inline .php-cs-fixer.dist.php:52]: If there are problems, let's fix rules if needed.

File: src/Config.php (hunk 1 above)
Comment (junichi11) [reply src/Config.php]: Fixed.

**Review by tmysik — APPROVED** [review]
Looks much better, thanks! :)

File: .php-cs-fixer.dist.php (hunk 2 above)
Comment (tmysik) [reply .php-cs-fixer.dist.php:52]: Thanks, looks great now!

**Pull request merged**